apis/storage/v1alpha1/snapshot_types.go

```diff
@@ -169,6 +169,14 @@ type Component struct {
 	// +optional
 	Phase ComponentPhase `json:"phase,omitempty"`
 
+	// Size represents the size of repository after last backup
+	// +optional
+	Size string `json:"size,omitempty"`
+
+	// Integrity represents the result of integrity check for the repository.
```

Comment:
// Integrity represents the result of the restic repository integrity check for this component